go/service/main.go

```diff
@@ -493,6 +493,11 @@ func (d *Service) writeServiceInfo() error {
 }
 
 func (d *Service) hourlyChecks() {
+	// do this right away
```

Comment:
should we replace this with a gregor notification?  i think we put these checks in before we had gregor.

Reply:
put in this check in `g.UserChanged`?

Reply:
There is a gregor notification, but if the app isn't running, you don't get it right away.